### PR TITLE
docs: 修改中英文的readme文档名，增加贡献指南英文版

### PR DESCRIPTION
**Pull request by fengyon**
# PR

rename: README.md              ->   README.zh-CN.md
rename: README.en.md         ->   README.md
rename: CONTRIBUTING.md  ->   CONTRIBUTING.zh-CN.md
add:       CONTRIBUTING.md

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-cli/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
+- Participate in code review.
 
-## 提交 Issue
+## Bug Reports
 
-如果你在使用 TinyCLI 组件过程中遇到问题，欢迎给我们提交 Issue，提交 Issue 之前，请先仔细阅读相关的[官方文档](https://opentiny.design)，确认这是一个缺陷还是尚未实现的功能。
+If you encounter problems in the process of using TinyCLI components, you are welcome to submit Issue to us. Before submitting Issue, please read the relevant [official documentation](https://opentiny.design) carefully to confirm whether this is a defect or an unimplemented function.
 
-如果是一个缺陷，创建新 Issue 时选择 [Bug report](https://github.com/opentiny/tiny-cli/issues/new?template=bug-report.yml) 模板，标题遵循 `[toolkitName/pluginName/CliCore]缺陷简述` 的格式，比如：`[tiny-toolkit-xxx] xxx套件执行tiny start命令启动后提示xxx错误`。
+If it is a defect, select [Bug report](https://github.com/opentiny/tiny-cli/issues/new?template=bug-report.yml) template when creating a new Issue. The title follows the format of `[toolkitName/pluginName/CliCore] defect description`. For example: `[tiny-toolkit-xxx] a xxx error was occurred after the 'tiny start' command was run by the xxx suite`.
```
Comment (fengyon) [inline CONTRIBUTING.md]: 这句翻译重点看一下

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
-- tiny-cli 和 node 的版本号
-- 缺陷的表现，可截图辅助说明，如果有报错可贴上报错信息
-- 缺陷的复现步骤，最好能提供一个最小可复现 demo 链接
+Issue that reports defects mainly needs to fill in the following information:
+- Version numbers of `tiny` and `node`.
+- The performance of the defect can be illustrated by screenshot, and if there is an error, the error message can be posted.
+- Defect reproduction step, preferably with a minimum reproducible demo link.
 
-如果是一个新特性，则选择 [Feature request](https://github.com/opentiny/tiny-cli/issues/new?template=feature-request.yml) 模板，标题遵循 `[toolkitName/pluginName/CliCore]新特性简述` 的格式，比如：`[CliCore] 新增switch命令支持切换NPM源功能`。
+If it is a new feature, select [Feature request](https://github.com/opentiny/tiny-cli/issues/new?template=feature-request.yml) template. The title follows the format of `[toolkitName/pluginName/CliCore] new feature description`. For example: `[CliCore] added the switch command to support switching of the NPM source`.
```
Comment (fengyon) [inline CONTRIBUTING.md:29]: 这句翻译重点看一下

**Comment by kagol** [comment]
微信号需要更新：opentiny -> opentiny-official

**Comment by fengyon** [comment]
已修改

**Review by kagol — APPROVED** [review]

**Comment by kagol** [comment]
@all-contributors please add @fengyon for doc

**Pull request merged**